src/safeds/data/tabular/plotting/_column_plotter.py

```diff
@@ -67,6 +66,68 @@ def box_plot(self) -> Image:
 
         return _figure_to_image(fig)
 
+    def violin_plot(self, theme: Literal["dark", "light"] = "light") -> Image:
```

Comment:
```suggestion
    def violin_plot(self, *, theme: Literal["dark", "light"] = "light") -> Image:
```

Parameter should be keyword-only.